tests/openvino/test_transform_fn.py

```diff
@@ -62,6 +64,12 @@ def multiple_inputs_as_dict_transform_fn(data_item):
     ],
 )
 def test_transform_fn(model, transform_fn, use_pot: bool):
+    if use_pot:
+        try:
+            importlib.import_module("openvino.tools.pot")
+        except ImportError:
+            pytest.skip("POT is not available in the current environment.")
+
```

Comment:
Why do not drop POT from this test as well?

Reply:
I suggest to do it in a separate PR where all pot references will be removed